README.md

```diff
@@ -2,25 +2,64 @@
 
 # operatorkit
 
-The operatorkit package implements an opinionated framework for developing
-Kubernetes operators. It can be used as a library in golang. It is built on
-top of our [microkit](https://github.com/giantswarm/microkit) framework which
-provides base functionality like logging and error handling.
+The operatorkit is a library for creating [Kubernetes operators][operators]. It
+emerged as extracted common functionality of number of the operators we
+developed in Giant Swarm. The goal of the library is to provide common
+structure of the operator projects and encapsulate best practices learned
+during running operators in production.
+
+## Features
+
+- Reducing boilerplate code.
+- CRD/TPR primitives allowing to reliably create, watch custom resources, and
+  decode custom objects.
+- Independent packages. It is possible to use only a part of the library. 
+- Possibility to change behaviour that often is specific to an organization
+  like logging and error handling.
+- Minimal set of dependencies.
 
 ## Current Scope
 
-The initial scope is intentionally small and will be expanded as the community
-gains more experience developing operators.
+Project is split into independent packages providing complementary
+functionalities allowing to create production grade Kubernetes operators.
 
-* Kubernetes clientset using [client-go](https://github.com/kubernetes/client-go) for accessing the K8s APIs.
+- client - provides unified way of creating Kubernetes clients required by
+  other packages.
+- crd/tpr - provide CRD/TPR primitives, allowing to reliably create custom
+  resources, wait for their initialization, and generate endpoints URLs. 
+- informer - provides well defined watching functionality for virtually any
```

Comment:
The aim for the informer is to be deterministic in how to apply the resync period and how to rate limit event processing. I would like to have this stated as well since this is the core idea behind building it on our own.

Reply:
This is what I wrote:

```
- informer - provides well defined watching functionality for virtually any
  Kubernetes resource. The informer is deterministic, meaning it does not
  dispatch events twice after the resync period, which saves some cycles. It
  also features rate limiting of the event dispatching. It also provides
  functionality for decoding custom objects, reducing error prone boilerplate
  code.
```

WDYT?

Reply:
I like this.